### PR TITLE
Add caramel snickerdoodle martini recipe

### PR DESCRIPTION
**Pull request by yoji-kojio**
Add caramel snickerdoodle martini recipe and fix some indents.

### CODE REVIEW

File: src/recipes/caramel-snickerdoodle-martini.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        {
+            "quantity": "",
+            "measure": "",
+            "ingredient": "cinnamon sticks, for garnish"
+        }
+    ],
+    "directions": [
+        "Place caramel on a plate and cinnamon sugar on another. Dip rims of martini glasses in caramel, then in cinnamon sugar. ",
+        "In a cocktail shaker, add heavy cream, Smirnoff Kissed Caramel, and cinnamon. Add ice and shake until cold.",
+        "Pour into glasses and top each one with whipped cream and a cinnamon stick."
```
Comment (kazuhirodk) [inline src/recipes/caramel-snickerdoodle-martini.json:50]: u could fragment some large steps :D haha
```suggestion
        "Place caramel on a plate and cinnamon sugar on another.",
        "Dip rims of martini glasses in caramel, then in cinnamon sugar. ",
        "In a cocktail shaker, add heavy cream, Smirnoff Kissed Caramel, and cinnamon.",
        "Add ice and shake until cold.",
        "Pour into glasses and top each one with whipped cream and a cinnamon stick."
```

File: src/recipes/chikoo-lassi.json [hunk 2]
```diff
@@ -18,31 +18,31 @@
             "measure": "cup",
             "ingredient": "milk"
         },
-		 {
+		{
```
Comment (kazuhirodk) [inline src/recipes/chikoo-lassi.json:21]: these indentations have extra spacing

File: src/recipes/chikoo-lassi.json [hunk 3]
```diff
@@ -18,31 +18,31 @@
...
             "quantity": "1/2",
             "measure": "teaspoon",
             "ingredient": "cardamom powder"
         },
-		 {
+		{
```
Comment (kazuhirodk) [inline src/recipes/chikoo-lassi.json:26]: these indentations have extra spacing

**Comment by alfg** [comment]
Thanks!

**Pull request merged**